prisma/migrations/20231026204718_uppercase_title/migration.sql

```diff
@@ -0,0 +1,191 @@
+-- AlterTable
```

Comment:
See the diff in the PR description to better understand the actual changes to these functions